loadtest/loadtest.go

```diff
@@ -14,13 +16,16 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	yaml "gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/helm/pkg/helm"
 )
 
 type Config struct {
 	ApprClient   apprclient.Interface
+	CPClients    *k8s.Clients
+	CPHelmClient *helmclient.Client
```

Comment:
The k8s job is created in the e2e cluster not the tenant cluster. This is to make it easier to debug the job.